src/test/librados/c_read_operations.cc

```diff
@@ -407,7 +407,7 @@ TEST_F(CReadOpsTest, Checksum) {
   }
 
   {
-    ceph_le32 init_value = init_le32(-1);
+    ceph_le32 init_value = ceph_le32(-1);
```

Comment:
we should probably stop assigning negative numbers to unsigned. Maybe use UINT*_MAX?

Reply:
will address this in a follow-up PR.